src/components/Modal/Modal.jsx

```diff
@@ -224,6 +243,10 @@ Modal.propTypes = {
    * Click outside modal will call onClose prop
    */
   clickToClose: PropTypes.bool,
+  /**
+   * Child elements (content) of modal when displayed
+   */
+  children: PropTypes.node,
```

Comment:
I had to add this for linting purposes.  Got a less confusing description?

Reply:
Nah, this looks good to me. :+1: